public/js/whiteboard.js

```diff
@@ -151,6 +153,9 @@ function clearDispatch() {
 }
 
 function handleDispatch(determinant, complaint, location) {
+  clearDispatch();
+  clearTimeout(dispatchTimeoutID);
```

Comment:
Shouldn't this be
```
if (dispatchTimeoutID != "")  {
    clearTimeout(dispatchTimeoutID);
    dispatchTimeoutId = "";
}
```
?

Reply:
That is most definitely more correct if not correct.

Reply:
> Shouldn't this be
> 
> ```
> if (dispatchTimeoutID != "")  {
>     clearTimeout(dispatchTimeoutID);
>     dispatchTimeoutId = "";
> }
> ```
> 
> ?

I've made the appropriate changes to reflect this corrected logic